src/util/Util.js

```diff
@@ -72,16 +72,20 @@ class Util {
    * * A URL-encoded UTF-8 emoji (no ID)
    * * A Discord custom emoji (`<:name:id>`)
    * @param {string} text Emoji string to parse
-   * @returns {Object} Object with `name` and `id` properties
+   * @returns {Object} Object with `animated`, `name` and `id` properties
    * @private
    */
   static parseEmoji(text) {
     if (text.includes('%')) text = decodeURIComponent(text);
     if (text.includes(':')) {
-      const [name, id] = text.split(':');
-      return { name, id };
+      const [, animated, name, id] = text.match(/(?:<(a)?:)?([\w_]{2,32}):(\d+)>?/);
+      return { animated: !!animated, name, id };
```

Comment:
Can you, for consistency with the rest of the lib, change the `!!animated` to `Boolean(animated)`?

Reply:
@SpaceEEC Done